apps/vaccitron/src/app/report-list-page/report-list-page.component.ts

```diff
@@ -46,9 +25,21 @@ export class ReportListPageComponent implements OnInit {
     this.#currentFilter = newFilter;
   }
 
-  constructor() {}
+  constructor(private vaccinesReportsService: VaccinesReportsService) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.getUpdate();
+    interval(environment.reportsQueryInterval)
```

Comment:
Good for now to trigger the interval internally. But if we do it that way, we need to push an update to the UI components in order for the UI to update, where is it?
(Or that it's some Angular's dependency property pattern that I don't know?)

Reply:
What I should actually do is create a service that handles that an exposes a Subjuect that listens to server updates or interval. Let's take this as tech debt and open a task for it? Or would you like it fixed now?

Reply:
Talked offline, as I suspected it's Binding - another type of dependency property that I don't know :)